.github/workflows/release.yaml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  release:
+    types: [created]
```

Comment:
I think this should get triggered on tag creation instead: https://github.com/jupyterhub/jupyter-remote-desktop-proxy/pull/46/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaL22

Releases are github specific, tags are universal to git!

Reply:
If we trigger on tag creation, do we expose ourselves to the risk that the latest release displayed on the GitHub landing page is out of sync with the latest release on pypi? (Which I think is our current situation on this repo.)

On -recipes, we release by clicking "make a new release" on GitHub, and then create the new tag as part of that process.

If this is triggered instead on tag, is the thought that the tags are still created on the GitHub browser UI? And then how do we keep GitHub releases in sync with tags... another step in this workflow that adds the release after the tag is made, following successful publish to pypi?

Reply:
I actually think nobody in the Jupyter project actually use releases, just tags - see https://github.com/jupyterhub/jupyterhub for example. 
<img width="205" alt="image" src="https://github.com/pangeo-forge/pangeo-forge-runner/assets/30430/6a5b6200-0872-4cba-a689-b6e9433670c4">

This made me realize I've never actually used "Releases" as a feature outside of tags. So I think my recommendation didn't actually have anything to do with github vs git, but just that I have no idea what the point of the "Releases" feature is! I know you can upload binary builds if you have 'em, but we don't. So I guess inherent in the suggestion to use tags is that - I don't know what the use case for GitHub Releases are, and I'm not sure why we would use that feature at all in any way or form! So there would actually be *fewer* steps with tags, as we would just not use Releases at all :D

Reply:
Haha! Okay this makes sense, tags it is 💫 😄